.github/mergify.yml

```diff
@@ -89,23 +91,19 @@ pull_request_rules:
       comment:
         message: |
           This PR is next in line to be merged, and will be merged as soon as checks pass.
-  - name: Notify author when merge queue failed
+  - name: Notify author on queue failure
     conditions:
-      - "check-failure=Queue: Embarked in merge train"
+      - 'check-failure=Queue: Embarked in merge train'
     actions:
       comment:
-        message: |
```

Comment:
Why remove this?

Reply:
It's replaced with another message that explains how to fix the problem.

Reply:
Oh sorry I meant why remove the "Nag changesets" message

Reply:
We already have the changesets reminder

Reply:
I see